README.adoc

```diff
@@ -625,6 +626,11 @@ https://www.linkedin.com/company/magnito-digital/[LinkedIn]
 |PHP, NodeJS, ReactJS, Block chain, Android, iOS
 |https://www.siliconorchard.com/[Website]
 
+|Singularity Limited
+|Level 5, House 147, Lane 1,Baridhara DOHS, Dhaka 1206, Bangladesh
+|Web and Mobile application,Virtual, Augmented and Mixed reality,Motion Graphics,Animation,Visual Effects
+|https://singularitybd.com/[Website]
+
```

Comment:
Can you add a space after a comma, in lines 630 and 631?

Reply:
done